public/css/calendar.less

```diff
@@ -324,6 +408,10 @@
         opacity: .8;
       }
     }
+
+    .description {
+      mix-blend-mode: screen;
```

Comment:
I've checked this with Florian again, and it seems to have no effect, not for the description nor for the title.